ci_framework/roles/repo_setup/tasks/main.yml

```diff
@@ -20,3 +20,6 @@
   ansible.builtin.include_tasks: configure.yml
 - name: Generate additional artifacts
```

Comment:
I think we shouldn't generate those artifacts if we get rhos-release. Maybe renaming "artifactss.yml" to "dlrn.yml" and include it only if `not cifmw_repo_setup_enable_rhos_release | bool` ?

Also, we probably don't want to install repo-setup if we're with rhos-release... ?

Reply:
So here is the workflow
1. for upstream, repo-setup tool will generate centos repos and dlrn repos in a directory.
2. for downstream, rhos-release tool will generate rhel base os repos and repo-setup tool will generate dlrn repos only for rhel (with the same command). 

So we need to install both rhos-release and repo-setup tool together.
artifactss.yml playbook is generate the dlrn md5 hash in both cases (upstream and downstream). So may be we should not rename it.

Reply:
hmmm ok. So it may be good to refactor this a bit in order to get all the generation of content in the artifacts.yml - this can be a follow-up, we can probably go as-is.